webpack.common.js

```diff
@@ -8,7 +8,10 @@ const PreloadWebpackPlugin = require('preload-webpack-plugin');
 
 module.exports = {
   mode: 'development',
-  entry: './src/index.js',
+  entry: {
+    main: './src/index.js',
+    vendor: './src/vendor.js'
```

Comment:
What if someone wants to use jQuery and another big framework that might conflict. They can't throw them both into `vendor.js` right? 

jQuery seems to give problems when trying to bundle via Webpack anyway, that's why I put the option to add externals in the bottom of the documentation.

I'll keep this main/vendor split though, I just want to understand this decision a little better.

Reply:
I split them into two separate files because vendor files don't change that often. e.g bootstrap js. so contentHash for vendor files won't be changed for vendor files as the content isn't changed. so those files can be cached by the browser. the browser won't have to fetch those files everytime and will use the cached files instead and most of the times vendor files are pretty large so if vendor chunk is cached. loading time will be reduced. and i have personally tested it by keeping bootstrap js which depends on jquery and popper js and it works fine.